src/styles/blocks/calendar.scss

```
@@ -0,0 +1,33 @@
+  html {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+
+  body {
+    justify-content: center;
+    display: flex;
+  }
+
+  .calendar {
+    width: 7 * ($days-width + $calendar-gap) + 20; //20 --- padding left + right
+    display: flex;
+    gap: 1px;
+    flex-wrap: wrap;
+    padding: 10px;
+
+  &__number {
+    transition-duration: 0.5s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```

---

transition: 0.5s в завданні